scripts/examples/mint_identifiers_cli.py

```diff
@@ -17,13 +17,14 @@
     "-u",
     "--url",
     type=str,
-    default="https://mars.cyverse.org/isamples_central/manage/mint_draft_datacite_identifiers",
+    default="https://central.isample.xyz/isamples_central/manage/mint_draft_datacite_identifiers",
     help="The url to the mint identifiers endpoint",
 )
 identifiers_option = click.option(
     "-n",
     "--num_identifiers",
     type=str,
+    default="1",
```

Comment:
Things totally break if this isn't specified, and the error message isn't helpful at all, so default it to 1.